src/less-style/structure.less

```diff
@@ -37,6 +37,23 @@ body.vellum-full-screen {
 
                 }
 
+                .fd-commander {
+                    display: flex;
+                    padding: 6px 4px 6px 12px;
+                    background-color: @cc-dark-cool-accent-mid;
+
+                    .fd-add-question-toggle {
+                        color: white;
+                        height: 17px;
+                    }
+
+                    input {
+                        border: 1px solid @cc-dark-cool-accent-low;
+                        flex: 2;
+                        margin: -2px 0;
```

Comment:
Would appreciate help with these styles. `flex` is only supported on modern browsers, which I think is fine, but happy to change if not recommended. Also the negative margin may not work well for other OS's text inputs (I only tested on Mac).